### DISCUSSION TITLE
Multiple schema.prisma files in one project

### DISCUSSION THREAD

**Discussion by arnavawasthi** [blitz-js/blitz#929] [Q&A] [answered]
I have two (actually more) dbs to connect from my prisma app. I could generate two clients (i could see them in node_modules/.prisma.) But for some reason, using these two clients in my app hasn't been successful. There have been two types of errors:
1. db model is not recognized
2. Or query engine for platform isn't found

I am using following commands to generate clients:
- npx prisma generate --schema=./db/schema.prisma
- npx prisma generate --schema=./db_xyz/schema.prisma
Is it supported to have multiple db connections in one app? Any help on this is much appreciated. Thanks.
Note: I am using Prisma with Blitz, so I got db folder by default and I created db_xyz manually for 2nd db.

I have followed this article to set this up: https://github.com/prisma/prisma/issues/2443#issuecomment-630679118

And following was the error after which I had no other option left but.

> Query engine binary for current platform "darwin" could not be found. This probably happens, because you built Prisma Client on a different platform. (Prisma Client looked in "/query-engine-darwin")
> You already added the platform "darwin" to the "generator" block in the "schema.prisma" file as described in https://pris.ly/d/client-generator  but something went wrong. That's suboptimal.
> Please create an issue at https://github.com/prisma/prisma-client-js/issues/new

**Comment by flybayer** [comment, marked as answer]
Hey @arnavawasthi sorry for the delayed reply here.

The query engine binary error shouldn't be related to the multiple schema files 🤔 

And from what you've described it sounds like this is an issue with Prisma, not the integration with Blitz. So I'm afraid I don't have any help to offer on this at the moment.

I suggest [joining the prism slack](https://prisma.slack.com) and posting this in there. Probably someone there can get you going!
